app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/RepeatPasscodeFragment.kt

```diff
@@ -67,18 +71,29 @@ class RepeatPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>(
                     }
                 }
                 is PasscodeViewModel.PasscodeSetup -> {
-                    if (ownerImported) {
-                        findNavController().popBackStack(R.id.ownerAddOptionsFragment, true)
+                    if (BiometricManager.from(requireContext())
```

Comment:
this can authenticate stuff is used in several places. What about extracting it to BiometricPasscodeManager